test/torch/tensors/test_precision.py

```diff
@@ -467,6 +466,34 @@ def test_torch_sigmoid_approx(workers):
 
             assert (diff / (tolerance * norm)) < 1
 
+def test_torch_tanh_approx(workers):
+    """
+    Test the approximate tanh with different tolerance depending on
+    the precision_fractional considered
+    """
+    alice, bob, james = workers["alice"], workers["bob"], workers["james"]
+
+    fix_prec_tolerance_by_method = {
+        "chebyshev": {3: 3 / 100, 4: 3 / 100, 5: 3 / 100},
```

Comment:
Those might need more tweaking - currently, it works, but we might want a lower bound

Reply:
That's ok I guess!